atlasdb-impl-shared/src/main/java/com/palantir/atlasdb/transaction/impl/SnapshotTransaction.java

```diff
@@ -447,7 +450,7 @@ public Iterator<Map.Entry<Cell, byte[]>> getRowsColumnRange(
         if (Iterables.isEmpty(rows)) {
             return Collections.emptyIterator();
         }
-        Iterable<byte[]> stableRows = ImmutableList.copyOf(rows);
+        ImmutableList<byte[]> stableRows = ImmutableList.copyOf(rows);
```

Comment:
For my curiosity, why is this called `stableRows`? 🐴

Reply:
The intention was probably to guarantee that the iteration order was fixed every time we iterate through it - I wondered who introduced it, and turns out it was me a while back: https://github.com/palantir/atlasdb/pull/4702/files

I'm not sure where I got the word from - it might be https://opendylan.org/books/drm/Iteration_Stability_and_Natural_Order or https://stackoverflow.com/questions/37227480/stable-element-ordering-issue-while-iterating-hashmap-in-java-8?